subxt/src/events/events_client.rs

```diff
@@ -96,7 +89,11 @@ where
     ) -> impl Future<
         Output = Result<EventSubscription<T, Client, EventSub<T::Header>>, Error>,
     > + Send
-           + 'static {
+           + 'static
+    where
+        Client: Send + Sync + 'static,
+        T: Send + Sync,
```

Comment:
I wonder why these additional bounds are needed?

(it's always a shame if we need any bounds on T, since T is a phantom type and it shouldn't matter it all. Perhaps we need to use PhantomDataSendSync in more places, which does a better job of ignoring T?)

Reply:
I compiler complained about the future returned by the event's client not being Send / Sync due to unsatisfied trait bounds on the market type `T`. `PhanthomDataSendSync` sounds like a viable option here, thanks!